raiden/routing.py

```diff
@@ -129,3 +163,49 @@ def get_best_routes(
         route_state = RouteState(partner_address, channel_state_id)
         available_routes.append(route_state)
     return available_routes
+
+
+def get_best_routes_pfs(
+        chain_state: ChainState,
+        token_network_id: typing.TokenNetworkID,
+        from_address: typing.InitiatorAddress,
+        to_address: typing.TargetAddress,
+        amount: int,
+        previous_address: typing.Optional[typing.Address],  # TODO(paul): Remove this?
+        config: Dict,
+) -> List[RouteState]:
+    pfs_path = '{}/api/v1/{}/paths'.format(
+        config['pathfinding_service_address'],
+        to_checksum_address(token_network_id),
+    )
+    payload = {
+        'from': to_checksum_address(from_address),
+        'to': to_checksum_address(to_address),
+        'value': amount,
+        'max_paths': config['pathfinding_max_paths'],
+    }
+    response = requests.get(pfs_path, params=payload)
+
+    if response.status_code != 200:
+        return []
+
+    paths = []
+    for path_object in response.json()['result']:
```

Comment:
This also needs error handling. The `.json()` call can fail if the response isn't well formed. And the response could be missing the `result` key if the server is having a fit.